src/main/java/org/folio/rest/validator/ResourcePutBodyValidator.java

```diff
@@ -67,7 +68,10 @@ private void validateManagedResourceIfNotSelected(ResourceDataAttributes attribu
     Boolean isHidden = attributes.getVisibilityData() != null ? attributes.getVisibilityData().getIsHidden() : null;
     EmbargoUnit embargoUnit = attributes.getCustomEmbargoPeriod() != null ? attributes.getCustomEmbargoPeriod().getEmbargoUnit() : null;
     List<Coverage> customCoverages = attributes.getCustomCoverages();
-    if (!isTitleCustom && (!Objects.isNull(cvgStmt) || !Objects.isNull(embargoUnit) || (!Objects.isNull(isHidden) && isHidden) || (!Objects.isNull(customCoverages) && !customCoverages.isEmpty()))) {
+    if (!isTitleCustom &&
+      (!Strings.isEmpty(cvgStmt) || !Objects.isNull(embargoUnit) ||
+      (!Objects.isNull(isHidden) && isHidden) ||
+      (!Objects.isNull(customCoverages) && !customCoverages.isEmpty()))) {
       throw new InputValidationException(INVALID_IS_SELECTED_TITLE, INVALID_IS_SELECTED_DETAILS);
```

Comment:
This LGTM @andrii-paias -- my main concern is the case where we deselect a resource -- we get errors from RM API if some of these fields are populated . This case should be caught here by validator

Reply:
yes, this should be caught by this condition, 
Here I only changed "!Objects.isNull(isHidden)" to "!Strings.isEmpty(cvgStmt)", RM API doesn't return an error if cvgStmt is an empty string, so we should let that case pass.